### ISSUE TITLE
[crazyscorner] Login failed: Selector

### ISSUE THREAD

**Issue by duindain** [Jackett/Jackett#7735]
### Environment

**OS**: Linux Mint 19.3

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono] Mono

**.Net Version**:

**Jackett Version**: 0.14.79.0

**Last Working Jackett Version**:

**Are you using a proxy or VPN?** [yes/no] no

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]
I've created a new account in crazyscorner
Checked mail messages and read them
entered password and username into jacket config for this tracker
clicked Okay

### Logged Error Messages

```
Jackett.Common.ExceptionWithConfigData: Login failed: Selector "a[href="logout.php"]" didn't match
   at Jackett.Common.Indexers.CardigannIndexer.TestLogin() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 810
   at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 969
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 70
```

[Make sure you attach the full logs with your personal information removed in case we need more information]

I've see this issue that was closed
https://github.com/Jackett/Jackett/issues/7146

I've checked my messages
Nothing seems to be outstanding and its a new account with no seeding issues since i haven't been able to connect through Jackett yet :)

**Comment by garfield69** [comment]
We will need to see the html that preceeds the error message to get a sense of what could be happening here.
Please provide the **full enhanced log** file. 
1. scroll down to the bottom of the Jackett Dashboard and *tick* the *enhanced Logging* checkbox
2. scroll up a bit and *click* on the *apply server settings* button
3. repeat the test
4. find the *log.txt* file (linux `~/.config/Jackett/`, windows `%ProgramData%\Jackett`)
5. edit it with a plain text editor, redacting any personal details, usernames, passwords, passkeys, hashes etc.
6. save, then drag-drop it here for us to take a look at.

(Note: When you restart Jackett the enhanced-log switch will be turned off. This prevents performance hits during normal operations).

Thanks.

**Comment by duindain** [comment]
I've upped the log level and re-attempted to add the indexer, I've attached the relevant parts from the log file

[Partial log file](https://rmu.no-ip.com/jackettmplog.txt)

**Comment by garfield69** [comment]
Your log shows that the URL that the Jackett indexer tried to use for logging in to the CrazyCorner web site was `http://crazys-corner.info/index.php?page=login`
this is the old URL, because on January 18th CrazyCorner updated their Web engine, and the correct URL to use for login is now `http://crazys-corner.info/takelogin.php`

Yet in your first post you stated you are using Jackett 0.14.79 Mono, which means that you should be using the correct version of the indexer that was made available since 0.12.1590 if everything was as it should be.

Which implies that for some reason your indexer did not get updated.

So I suggest you manually install the latest Mono version from https://github.com/Jackett/Jackett/releases/download/v0.14.136/Jackett.Binaries.Mono.tar.gz

and that should solve you issue.

BTW, we recommend ditching the Mono dependent version of Jackett in favour of the .Net Core version of Jackett, if that is at all possible.
You would need to be on Linux Mint 17 or 18 using an x64 processor.
If this is the case, then consider installing using the installation notes from https://github.com/Jackett/Jackett#install-on-linux-amdx64

let me know how you go with whatever install you decide to use.

**Comment by duindain** [comment]
Thanks @garfield69 I've updated to the .net core version that was released a few hours ago and deleted my old version

Seems to have detected the changed url this time and its connected fine

Thanks for your help :)
